desktop/core/src/desktop/manage_entry.py

```diff
@@ -85,11 +85,8 @@ def entry():
 
   if len(sys.argv) > 1:
     subcommand = sys.argv[1]
-  if sys.version_info[0] < 3:
-    args = [None]
-  else:
-    args = []
-  parser = CommandParser(*args, usage="%(prog)s subcommand [options] [args]", add_help=False)
+
+  parser = CommandParser(None, usage="%(prog)s subcommand [options] [args]", add_help=False)
```

Comment:
I think We want Django 1.11 and Django 2.1 ( i.e. 1.11 and the latest one) why you changed that? @romainr